db/schema_tables.cc

```diff
@@ -1697,6 +1697,13 @@ static std::vector<data_value> read_arg_values(const query::result_set_row& row)
     }
 #endif
 
+static std::pair<cql3::functions::function_name, std::vector<data_type>> drop_func(replica::database& db, const query::result_set_row& row) {
```

Comment:
`struct function_signature { ... }`

Reply:
This should not be necessary anymore in the last rebase.